### PR TITLE
Update CMake build

### PR DESCRIPTION
**Pull request by Patater**
Update the example to work with the current Mbed OS CMake support.

### CODE REVIEW

**Comment by LDong-Arm** [comment]
Looks like Travis is not using `.travis.yml` from this PR. It's worth checking the setting in https://travis-ci.com/github/ARMmbed/mbed-os-example-sys-info which I don't have permissions for.
@0xc0170 Are you able to help?

File: README.md [hunk 1]
```diff
@@ -18,10 +18,18 @@ The `main()` function outputs on the serial interface information about the hard
 1. Connect a USB cable between the USB port on the target and the host computer.
 1. Run the following command to build the example project and program the microcontroller flash memory:
 
-   ```bash
-   $ mbed compile -m <TARGET> -t <TOOLCHAIN> --flash --sterm
-   ```
-   
+    * Mbed CLI 2
```
Comment (LDong-Arm) [inline README.md:30]: Please also update the first section above regarding Mbed CLI. We could use https://github.com/ARMmbed/mbed-os-example-blinky#mbed-os-build-tools as a template.

**Review by LDong-Arm — CHANGES_REQUESTED** [review]

**Review by LDong-Arm — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -18,38 +27,59 @@ The `main()` function outputs on the serial interface information about the hard
...
+    $ mbed-tools sterm
+    ```
 
-``` 
+* Mbed CLI 1
+    ```bash
+    $ mbed sterm
+    ```
+
+Depending on the target, you can build the example project with the `GCC_ARM`, `ARM` or `IAR` toolchain. After installing Arm Mbed CLI, run the command below to determine which toolchain supports your target:
```
Comment (LDong-Arm) [inline README.md]:
```suggestion
Depending on the target, you can build the example project with the `GCC_ARM` or `ARM` toolchain. After installing Arm Mbed CLI, run the command below to determine which toolchain supports your target:
```

**Review by LDong-Arm — CHANGES_REQUESTED** [review]

**Review by LDong-Arm — APPROVED** [review]
LGTM

**Review by 0xc0170 — APPROVED** [review]

**Pull request merged**